server/controllers/contest.js

```diff
@@ -0,0 +1,58 @@
+const Contest = require("../models/Contest");
+const asyncHandler = require("express-async-handler");
+
+exports.createContest = asyncHandler(async (req, res) => {
+    try {
+        // expecting req.body to come back in this format.
+        // {
+        // title: req.body.title,
+        // description: req.body.description,
+        // prizeAmount: req.body.prizeAmount,
+        // deadlineDate: req.body.deadlineDate,
+        // not positive how the userid is going to come from the front-end, leaving req.body for now, will change based on how id is being sent from front to back. 
+        // userId: req.body.userId
+        // }
+        const contest = await Contest.create(req.body);
+        res.status(200).json(contest);
```

Comment:
Please update status code to 201 here - for successful resource creation and update it 500 in the catch block - for a server error :)

Please see for reference: https://www.restapitutorial.com/httpstatuscodes.html

Reply:
absolute brain fart.